### PR TITLE
[2.11.x] DDF-3502 Clarify documentation for setting up a highly available cluster

### PR DESCRIPTION
**Pull request by Lambeaux**
#### What does this PR do?
Adds clarification for certain HA setup steps. 

#### Who is reviewing it? 
@brjeter 
@blen-desta 
@oconnormi 

#### Choose 2 committers to review/merge the PR. 
@coyotesqrl
@shaundmorris

#### How should this be tested?
Verify documentation. 

#### Any background context you want to provide?
Feedback was generated from third party review and attempted setup of an HA cluster. 

#### What are the relevant tickets?
[DDF-3502](https://codice.atlassian.net/browse/DDF-3502)

#### Checklist:
- [x] Documentation Updated

#### Screenshots (if appropriate)
image

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc [hunk 1]
```diff
@@ -55,8 +57,31 @@ One way to accomplish this is to use the default <<org.codice.ddf.catalog.conten
...
+
+[NOTE]
+====
+Systems that use DDF version 2.11.4 or later will automatically get a DNS SAN entry matching the CN
+without the need to specify the `-san` argument to the `CertNew` command.
+====
+
+More customization for certs can be achieved by following the steps at
+<<_creating_new_server_keystore_entry_with_the_certnew_scripts,Creating New Server Keystore Entry with the CertNew Scripts>>.
 This certificate is intended for testing purposes only, and should not be used in production.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc]: Can we maybe move this sentence up to the beginning of the self-signed certificates? To make it more prominent?

File: distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc [hunk 2]
```diff
@@ -9,24 +9,26 @@
 
 These steps are handled differently for the initial setup of a Highly Available Cluster.
 
-===== HTTP Proxy Integration
+===== Failover Proxy Integration
 
-In order to integrate with an HTTP proxy, the ${branding} node's system properties (in `${home_directory}/etc/system.properties`) must be changed to publish the correct port to external systems and users.
-This must be done before installing the first ${branding} node.
+In order to integrate with an failover proxy, the ${branding} node's system properties (in `${home_directory}/etc/system.properties`) must be changed to publish the correct port to external systems and users.
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc]: ❗️ s/b "...with a failover proxy..."

File: distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc (hunk 1 above)
Comment (coyotesqrl) [reply distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc]: Would a `[NOTE]` be better?

File: distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc (hunk 2 above)
Comment (coyotesqrl) [reply distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc]: ping, @Lambeaux

File: distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc (hunk 1 above)
Comment (Lambeaux) [reply distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc]: 👍  to both comments.

**Comment by Lambeaux** [comment]
Build now.

**Comment by Lambeaux** [comment]
Did a quick manual hero. Opened docs. All changes are rendering correctly and links work appropriately. Screen shot added to description.

**Pull request merged**